### ISSUE TITLE
pip install error for pyNN

### ISSUE THREAD

**Issue by ashwin4ever** [NeuralEnsemble/PyNN#666]
I am unable to install using pip on a windows 10 machine and get the following error
```
ERROR: The tar file (C:\Users\userx\AppData\Local\Temp\pip-unpack-f4dndy7x\PyNN-0.9.5.tar.gz) has a file (C:\Users\userx\AppData\Local\Temp\pip-install-pch3k8lf\PyNN\pyNN/hardware/aux.py) trying to install outside target directory (C:\Users\userx\AppData\Local\Temp\pip-install-pch3k8lf\PyNN)

```
I tried the manual install as well which does not proceed beyond a point.

**Comment by Lxnus** [comment]
Hey.

May that will work for you

```
pip install sPyNNaker==1!1.5.10

```

**Comment by tollybe** [comment]
This was also an issue for me.  

`ERROR: The tar file (C:\Users\user\AppData\Local\Temp\pip-unpack-zydj1cpr\PyNN-0.9.5.tar.gz) has a file (C:\Users\user\AppData\Local\Temp\pip-install-httbe87q\pyNN\pyNN/hardware/aux.py) trying to install outside target directory (C:\Users\user\AppData\Local\Temp\pip-install-httbe87q\pyNN)`


I  believe it's the ".\PyNN/hardware/aux.py" is the reason as this is a unix /linux path. I fixed it by running the manual install with no problems:

1. installing on desktop: https://pypi.org/project/PyNN/#files 
2. Unzipping and then navigate using cd C:\Users\user\Desktop\PyNN-0.9.5\PyNN-0.9.5>pip3 setup.py install
(3.  if you're running it for spynnaker: rerun the pip3 install sPyNNaker8)

Hope it helps.

**Comment by ChihweiLHBird** [comment]
I also got this issue. Wondering if there is a fix in progress.

**Comment by apdavison** [comment]
This is essentially a duplicate of #446, and is fixed by #678
